### ISSUE TITLE
Android: how to store and use images in xml file of android studio. 

### ISSUE THREAD

**Issue by kritika2311** [girlscript/winter-of-contributing#74]
1. Will describe where to store the image which we have to use for our project.
2. how can we refer the image in our xml file.

**Comment by kritika2311** [comment]
@VikasPandey121 can I work on this issue.

**Comment by VikasPandey121** [comment]
Please keep these steps in mind while creating a pull request:

- Your Contributing file should be in this directory `Android_Development/Java/` with the file name starting from 1.7 for instance `1.7 Store and use images in XML`.
Note:- If you have multiple files to commit then add a tag at the beginning of your file name like `1.7.1`, `1.7.2` etc
- Create a new branch and push your PR on `Android_Development_With_Java` branch.
- Pull request should contain only one commit
Note: If you have multiple commits then `squash` your commit.

**Comment by VikasPandey121** [comment]
> @VikasPandey121 can I work on this issue.

Definitely @kritika2311, You are good to go. I am assigning this issue to you.

**Comment by SouravKr2001** [comment]
I am interested in solving this issue (documentation)

**Comment by VikasPandey121** [comment]
> I am interested in solving this issue (documentation)

It has been already assigned to another contributor. Please contributor to any other issue which is still yet to assign.
PS: get the issue assigned first.

**Comment by kritika2311** [comment]
Created my pull request for this issue please verify it @VikasPandey121 sir.
Link - https://github.com/girlscript/winter-of-contributing/pull/1205#issue-737388119

**Comment by Rohit-2602** [comment]
@kritika2311  can you please give a progress update about the issue and if you are facing any problem feel free to ask.

**Comment by kritika2311** [comment]
@Rohit-2602 sir due to exams, I was not active for a period. I will resolve this issue in a day or two .

**Comment by kritika2311** [comment]
And sir @Rohit-2602  I am not able to understand how to make the pull request by creating the new branch, please help.

**Comment by Rohit-2602** [comment]
@kritika2311 see discord pinned messages

**Comment by nikki-priyaHIT** [comment]
may I work on this issue if no one have assigned?
Nikki Priya
Open Source Batch- 1
Documentation

**Comment by VikasPandey121** [comment]
Yes sure @nikki-priyaHIT, I am assigning this issue to you as @kritika2311 has not provided any updates regarding this PR.
